incubator/nodejs/image/project/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "nodejs",
+  "version": "0.2.0",
```

Comment:
This should be 0.2.1 to keep it in line with the stack version

Reply:
Done